lib/karaokium_web/views/layout_view.ex

```diff
@@ -29,30 +29,37 @@ defmodule KaraokiumWeb.LayoutView do
   defp admin_menu(conn) do
     [
       %{
-        title: "Events",
+        title: "Options",
         url: "#",
         submenu: [
           %{title: "Karaokes", url: Routes.admin_karaoke_index_path(conn, :index)},
-          %{title: "Locations", url: Routes.admin_location_index_path(conn, :index)}
-        ]
-      },
-      %{title: "Teams", url: Routes.admin_team_index_path(conn, :index), submenu: []},
-      %{
-        title: "Repertoire",
-        url: "#",
-        submenu: [
+          %{title: "Locations", url: Routes.admin_location_index_path(conn, :index)},
           %{title: "Search Songs", url: Routes.admin_song_search_path(conn, :new)},
           %{title: "Songs", url: Routes.admin_song_index_path(conn, :index)},
           %{title: "Artists", url: Routes.admin_artist_index_path(conn, :index)},
-          %{title: "Albums", url: Routes.admin_album_index_path(conn, :index)}
+          %{title: "Albums", url: Routes.admin_album_index_path(conn, :index)},
+          %{title: "Teams", url: Routes.admin_team_index_path(conn, :index)}
         ]
       }
     ]
   end
 
   defp sysadmin_menu(conn) do
     [
-      %{title: "LiveDashboard", url: Routes.live_dashboard_path(conn, :home), submenu: []}
```

Comment:
A sysadmin of the system should only get permission to access the Live Dashboard in order to monitor the application.

Reply:
Ok. I am commiting it to another branch since this branch is for register_page and not for navbar.